modules/DMZ/project_config/README.md

```diff
@@ -79,14 +141,20 @@ The variables the program can access from this object are:
 
 - `models`: A dictionary of Model objects from the `<Models>` tag where the _key_ is the name of the model and the _value_ is the `Model` object.
 
-- `test_files`: A list of file names to be used for testing. Comes from the `<Files>` tag where the `<File>` attribute `type` equals "test".
+- `test_files`: A list of file names to be used for testing. Comes from the `<Files>` tag where the `<File>` or `<Folder>` attribute `type` equals "test".
 
 
-- `train_files`: A list of file names to be used for training. Comes from the `<Files>` tag where the `<File>` attribute `type` equals "train".
+- `train_files`: A list of file names to be used for training. Comes from the `<Files>` tag where the `<File>` or `<Folder>` attribute `type` equals "train".
 
 #### API
 - `save(filename=config_file_name)`: Saves the current configuration to XML. By default uses the same name as what was passed in.
 

```

Comment:
I think these below need a space, as they aren't currently a bulleted list according to my viewer (Doc viewer? That thingy.)